File: app/src/main/java/com/cr4sh/nhlauncher/Overrides/FastScrollRecyclerView.java
package com.cr4sh.nhlauncher.Overrides;

import android.content.Context;
import android.util.AttributeSet;

import androidx.annotation.Nullable;
import androidx.recyclerview.widget.RecyclerView;

public class FastScrollRecyclerView extends RecyclerView {

    public FastScrollRecyclerView(Context context) {
        super(context);
        init();
    }

    public FastScrollRecyclerView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
        init();
    }

    public FastScrollRecyclerView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        init();
    }

    private void init() {
        // Attach custom SnapHelper on init
        NHLSnapHelper snapHelper = new NHLSnapHelper();
        snapHelper.attachToRecyclerView(this);
    }
}

File: app/src/main/java/com/cr4sh/nhlauncher/overrides/FastScrollRecyclerView.kt
package com.cr4sh.nhlauncher.overrides

import android.content.Context
import android.util.AttributeSet
import androidx.recyclerview.widget.RecyclerView

class FastScrollRecyclerView : RecyclerView {
    constructor(context: Context?) : super(context!!) {
        init()
    }

    constructor(context: Context?, attrs: AttributeSet?) : super(context!!, attrs) {
        init()
    }

    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context!!,
        attrs,
        defStyleAttr
    ) {
        init()
    }

    private fun init() {
        // Attach custom SnapHelper on init
        val snapHelper = NHLSnapHelper()
        snapHelper.attachToRecyclerView(this)
    }
}
